### DISCUSSION TITLE
Blood? Gore? Links to unmoderated chats? livestreams?  chats made with code?

### DISCUSSION THREAD

**Discussion by CloudGate1234** [NovaoTech/solaris#13] [Ideas]
policies about things in title.

**Comment by MikeDev101** [comment]
1. Peer-to-peer, private chats should be allowed. However, trying to get a large group of people on it (aka publically advertising a group chat) shouldn't be allowed.
2. Livestreams are risky, but if it's a voice-only live stream and with no face cam/personal info displayed, it should be ok.
3. Chats made with code should be accepted, but we need to have a publically available (everyone can use it) and contributable (everyone should be able to improve it, a suggest-and-review type of system) whitelist.
4. Blood and gore should be allowed but within reason. First suggestion: We have a toggle setting that can enable/disable search results containing projects that have blood/gore. Another: Filter all projects containing the sorts on the first page, unless explicitly enabled.

**Comment by icarusinthewings** [comment]
We will have community guidelines to prevent profanity and anything more than light blood and gore. Additionally, all extensions can only be sourced from the Polaris marketplace. Anyone can submit an extension to the marketplace but it will be reviewed by a staff member in up to 72h. I might introduce a "Polaris Plus" that allows users to get faster review times. Chat projects will be disallowed due to the inclusion of Opt-in messages that will be filtered through an AI bot. Additionally, Staff members will be able to view these chats. Although this can be intrusive, we will have a strict policy of having no chat searching without warrant. Live streams are disallowed.
